### ISSUE TITLE
Rare Lag / bump for image files when using page indicator / mouse scrolling

### ISSUE THREAD

**Issue by omodia** [mienaiyami/yomikiru#155]
### Setup or portable version of the app?

Portable

### Steps to reproduce

- [Chapter 1.zip](https://github.com/mienaiyami/yomikiru/files/11518424/Chapter.1.zip)
- [Chapter 2.zip](https://github.com/mienaiyami/yomikiru/files/11518443/Chapter.2.zip)
- [Expected Behavior.zip](https://github.com/mienaiyami/yomikiru/files/11518423/Expected.Behavior.zip)

1. Extract the above zip files and open them - For better experience and faster loading for comparison.
2. Type a value in the page indicator for the expected behavior file above.
3. Now, type a value in the page indicator for the chapter 1, chapter 2.
4. Compare them both.
5. You can also use the scroll bar for comparison, drag the scroll bar down.

https://github.com/mienaiyami/yomikiru/assets/116664905/b8815f7e-4d46-4ba0-be41-c546b9798aa7

### To be as smooth as the below test file with no lags / bumps while using page indicator, scroll bar.
---

1. Open expected behavior.zip and extract it to a folder.
2. Copy and paste upto 1600 files.
3. Now navigate using page number.
4. It is still smooth with no lag at all, even when the total folder size is 255 MB with 1600 images.
5. In layman terms, I am torture testing the app, but still it only uses 5 - 7% of CPU usage while fast scrolling through 1000's of images with no lags at all.

https://github.com/mienaiyami/yomikiru/assets/116664905/f7eb92a9-8e18-4689-8f7d-16de37970a22

### 2. Rarely, lags / bumps also occur while scrolling.
---

Bumps / Lags related,

1. I tested the mouse scrolling for 15 times on these two files, it only had lags for 2 times out of 20 times while using scrolling.
2. That too not for the entire chapter, but for 5 - 7 images.
3. I opened the dev tools to see if there is any error, but unlike the pervious versions there is no decode error too.

https://github.com/mienaiyami/yomikiru/assets/116664905/66dbd217-7cfe-4d4b-9b48-152750e0ae67

https://github.com/mienaiyami/yomikiru/assets/116664905/d5c845e1-916d-4031-a474-e02d9e26e292

https://github.com/mienaiyami/yomikiru/assets/116664905/394d3872-f208-4cf1-ab7d-ef7df58fe6b5

**Comment by mienaiyami** [comment]
Thanks, I will look into it. 
And maybe also add a very small delay before changing page on input to prevent unnecessary scrolling.

**Comment by mienaiyami** [comment]
The above error is probably related to shortcut saving. Did you change shortcut settings?
I have fixed that and will be releasing on Saturday or Sunday.

**Comment by mienaiyami** [comment]
It cant be fixed. It is possibly system related.

**Comment by mienaiyami** [comment]
I cant control resources allocated to app. It is controlled by the framework.

**Comment by mienaiyami** [comment]
Its actually normal. Image resolution is maybe too high or is it webp? Its hard to control everything through provided methods.
I believe it has to do more with this specific image creator, i think it is the same manga which started the old stuttering issue.

**Comment by mienaiyami** [comment]
What do you mean by this? Does this have something to do with webp?

I meant the images from that studio. And yes, webp images are more compressed than jpg and png.

**Comment by mienaiyami** [comment]
It is also related to #140 and #143. Things above will not cant any measuresable difference. 
I think I will need to add option to toggle hardware acceleration.
